File: desktop/src/com/rartworks/engine/desktop/GooglePlayServicesMock.java
package com.rartworks.engine.desktop;

import com.rartworks.engine.apis.GooglePlayServices;

public class GooglePlayServicesMock implements GooglePlayServices {
	@Override
	public void signIn() {

	}

	@Override
	public void signOut() {

	}

	@Override
	public void rateGame() {

	}

	@Override
	public void submitScore(long score) {

	}

	@Override
	public void showScores() {

	}

	@Override
	public void unlockAchievementByScore(int score) {

	}

	@Override
	public boolean isSignedIn() {
		return false;
	}
}

File: desktop/src/com/rartworks/engine/desktop/GooglePlayServicesMock.kt
package com.rartworks.engine.desktop

import com.rartworks.engine.apis.GooglePlayServices

class GooglePlayServicesMock : GooglePlayServices {
	override fun signIn() {

	}

	override fun signOut() {

	}

	override fun rateGame() {

	}

	override fun submitScore(score: Long) {

	}

	override fun showScores() {

	}

	override fun unlockAchievementByScore(score: Int) {

	}

	override fun isSignedIn(): Boolean {
		return false
	}
}
